code/modules/client/asset_cache.dm

```diff
@@ -309,6 +309,20 @@ GLOBAL_LIST_EMPTY(asset_datums)
 /datum/asset/chem_master/send(client)
 	send_asset_list(client, assets, verify)
 
+//Cloning pod sprites for UIs
+/datum/asset/cloning
+	var/assets = list()
+	var/verify = FALSE
+
+/datum/asset/cloning/register()
+	assets["pod_idle.gif"] = icon('icons/obj/cloning.dmi', "pod_idle")
+	assets["pod_cloning.gif"] = icon('icons/obj/cloning.dmi', "pod_cloning")
+	assets["pod_mess.gif"] = icon('icons/obj/cloning.dmi', "pod_mess")
+	for(var/asset_name in assets)
```

Comment:
*Not a requested change*

This will automatically base64 things? Good to know.

Reply:
Technically it uses `browse_rsc` - my original plan was to make these icons animated on the UI but it seems that they stop being animated when cached, so that plan was quickly shut down